testing/ssh-honeypot/10-Makefile.patch

```diff
@@ -0,0 +1,15 @@
+--- a/Makefile.orig
++++ b/Makefile
+@@ -1,9 +1,10 @@
+ CC=clang
+ CFLAGS=-Wall -static-libgcc
+ LIBS=-lssh -ljson-c
++DESTDIR:=
+ 
+ ssh-honeypot:
+-	$(CC) $(CFLAGS) -o bin/ssh-honeypot src/ssh-honeypot.c $(LIBS)
++	$(CC) $(CFLAGS) -o $(DESTDIR)/bin/ssh-honeypot src/ssh-honeypot.c $(LIBS)
```

Comment:
Seems weird to build and install here, just remove this patch and install in the APKBUILD